src/endpoint/blob/blob_utils.js

```diff
@@ -64,12 +64,13 @@ function parse_copy_source(req) {
 async function list_objects(params, account_name, container, sasToken) {
 
     const hostname = `${account_name}.blob.core.windows.net`;
-    const delimiter = params.delimiter || '/';
     const maxResults = params.limit === 0 ? 1 : params.limit;
 
-    let path = `https://${account_name}.blob.core.windows.net/${container}?restype=container&comp=list&delimiter=${delimiter}` +
+    let path = `https://${account_name}.blob.core.windows.net/${container}?restype=container&comp=list` +
         `&maxresults=${maxResults}&${sasToken}`;
```

Comment:
Maybe remove maxresults as well? 
It is optional and has a default of 5000.
I think that it is better to handle it like all others below.

Reply:
maxResults get a value of 1000 in s3_get_bucket.js for consistency with AWS, I think we should keep it for not getting 5000 records and to get 1000 records as default instead.

Reply:
It is also a confusing factor for someone that wants to use the Azure NS only and sends requests for 5000 blobs but gets up to 1000.

Reply:
so should we be compatible with AWS or Azure?
return by Default 1000 records or 5000?